processing/src/main/java/org/apache/druid/query/expression/TimestampExtractExprMacro.java

```diff
@@ -73,38 +75,39 @@ public Expr apply(final List<Expr> args)
       throw validationFailed("unit arg must be literal");
     }
 
-    if (args.size() > 2) {
-      validationHelperCheckArgIsLiteral(args.get(2), "timezone");
+    Optional<DateTimeZone> timeZone = Optional.empty();
+    if (args.size() > 2 && args.get(2).isLiteral()) {
+      timeZone = Optional.of(ExprUtils.toTimeZone(args.get(2)));
     }
 
-    final Expr arg = args.get(0);
```

Comment:
you could still keep it.

Reply:
no reference for this arg, so I removed it.